src/sdk/data/data.module.ts

```diff
@@ -249,4 +250,96 @@ export class DataModule {
       throw new Error(error.message || 'Failed to fetch exchange rates');
     }
   }
+
+  async getSupportedAssets(chainId?: number, provider?: QuotesProvider): Promise<Token[]> {
```

Comment:
`QuotesProvider` is an ambiguous name. What we are getting here are offers/orders, so we have call them accordingly. In this case, they are bridges, so we can name them BridgingProvider